repo/packages/N/nexus/0/config.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
Its just bad UX when when a user clicks on `install` in Universe. We may be able to improve the workflow in future. I understand the default installation wouldn't be used for any real use case but thats what the `example` is for. 

I get your point. Lets keep the way it is now and I think you've added that info in the example?